sklearn/discriminant_analysis.py

```diff
@@ -425,20 +425,6 @@ def fit(self, X, y, store_covariance=None, tol=None):
         y : array, shape (n_samples,)
             Target values.
         """
-        if store_covariance:
```

Comment:
Investigating any leftovers from this, I've discovered that QDA has `store_covariances` unlike LDA's `store_covariance` :((